scripts/DeployAndAddTracer.js

```diff
@@ -1,67 +1,54 @@
-const hre = require("hardhat")
-const TestToken = artifacts.require("TestToken")
-const { BN, time } = require("@openzeppelin/test-helpers")
-const Gov = artifacts.require("Gov")
-const TracerFactory = artifacts.require("TracerPerpetualsFactory")
+const hre = require("hardhat");
+const { BN } = require("@openzeppelin/test-helpers");
```

Comment:
Does `ethers` not provide its own `BigNumber` implementation?

Reply:
this was just legacy, ethers does ill use that instead